engine/install/debian.md

```diff
@@ -31,13 +31,19 @@ architectures.
 
 ### Uninstall old versions
 
-Older versions of Docker went by the names of `docker`, `docker.io`, or
-`docker-engine`, you might also have installations of `containerd` or `runc`.
-Uninstall any such older versions before attempting to install
-a new version:
+Older versions of Docker were called `docker`, `docker.io`, and
+`docker-engine`. Make sure that such old versions of Docker aren't
+installed on your system before you install a current version.
+
+Docker Engine depends on `containerd` or `runc`. Likewise, if you have
+those packages installed previously, you may want to uninstall them to avoid
+conflicts.
```

Comment:
We need both `containerd` and `runc`, but we bundle them in `containerd.io`, which can conflict with the distro's `containerd` and `runc` packages.